internal/operands/template-validator/reconcile.go

```diff
@@ -137,6 +138,10 @@ func reconcileService(request *common.Request) (common.ResourceStatus, error) {
 func reconcileDeployment(request *common.Request) (common.ResourceStatus, error) {
 	validatorSpec := request.Instance.Spec.TemplateValidator
 	image := getTemplateValidatorImage()
+	if image == "" {
+		request.Logger.Info(fmt.Sprintf("VALIDATOR_IMAGE Envirnoment variable: %s", os.Getenv(common.TemplateValidatorImageKey)))
```

Comment:
If `image == ""` the env var `VALIDATOR_IMAGE` will always be `""`, so this log line is not needed.

Reply:
Removed